src/sentry/testutils/factories.py

```diff
@@ -1510,6 +1511,11 @@ def create_notification_action(
 
         return action
 
+    @staticmethod
+    @assume_test_silo_mode(SiloMode.REGION)
```

Comment:
is it possible to hit control silo endpoints with a basic auth header?

Reply:
Actually since this is only formatting and not creating any objects, I should remove that decorator alltogether